tests/baselines/reference/declarationEmitObjectLiteralAccessorsJs1.js

```diff
@@ -0,0 +1,80 @@
+//// [tests/cases/compiler/declarationEmitObjectLiteralAccessorsJs1.ts] ////
+
+//// [index.js]
+// same type accessors
+export const obj1 = {
+  /**
+   * my awesome getter (first in source order)
+   * @returns {string}
+   */
+  get x() {
+    return "";
+  },
+  /** 
+   * my awesome setter (second in source order)
+   * @param {string} a
+   */
+  set x(a) {},
+};
+
+// divergent accessors
+export const obj2 = {
+  /** 
+   * my awesome getter
+   * @returns {string}
+   */
+  get x() {
+    return "";
+  },
+  /** 
+   * my awesome setter
+   * @param {number} a
+   */
+  set x(a) {},
+};
+
+export const obj3 = {
+  /**
+   * my awesome getter
+   * @returns {string}
+   */
+  get x() {
+    return "";
+  },
+};
+
+export const obj4 = {
+  /**
+   * my awesome setter
+   * @param {number} a
+   */
+  set x(a) {},
+};
+
+
+
+
+//// [index.d.ts]
+export namespace obj1 {
+    let x: string;
+}
+export const obj2: {
+    /**
+     * my awesome getter
+     * @returns {string}
+     */
+    get x(): string;
+    /**
+     * my awesome setter
+     * @param {number} a
+     */
+    set x(a: number);
+};
+export namespace obj3 {
+    let x_1: string;

```

Comment:
This is going to change because I merged your other jsdoc declaration emit PR, right? I think this is ready to go once it's sync'd, though - perf did come back clean, after all.

Reply:
Yep, this was impacted by that other PR. I just synced with `main` so this should now be mergeable.